packages/pageindicator/pageindicator.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import { html, LitElement } from 'lit';
+import './index.ts';
+
+export default {
+  component: 'w-pageindicator',
+  title: 'PageIndicator',
```

Comment:
Another nit, feels like this could be grouped with Pagination and such in the sidebar
```suggestion
  title: 'Navigation/PageIndicator',
```